templates/terraform/environments/prod/user_access.tf

```diff
@@ -1,3 +1,13 @@
+locals {
+  # user-auth:enabled will allow deployer to manage oathkeeper rules, otherwise concats with []
+  auth_enabled = <% if eq (index .Params `userAuth`) "yes" %>true<% else %>false<% end %>
+  auth_deploy_rules = local.auth_enabled ? [{
+      verbs      = ["get", "create", "delete", "patch", "update"]
+      api_groups = ["*"]
```

Comment:
i tried `oathkeeper.ory.sh/v1alpha1` and `oathkeeper.ory.sh` as the api group, but didnt work 🤔